### PR TITLE
add AutomaticEnv support

### PR DESCRIPTION
**Pull request by manjunathb4461**
- Add support to read vars from environment
- Add support to bind them to config object

### CODE REVIEW

File: venomoid.go [hunk 1]
```diff
@@ -116,3 +121,18 @@ func (c *ConfigBuilder) WithErrorOnMissing(eom bool) *ConfigBuilder {
 	c.errorOnMissingFile = eom
 	return c
 }
+
+func (c *ConfigBuilder) WithAutomaticEnv(automaticEnv bool) *ConfigBuilder {
+	c.automaticEnv = automaticEnv
+	return c
+}
+
+func BindEnv(input ...string) error {
```
Comment (ranglust) [inline venomoid.go]: why is this function required? what's the behavior if the user does not execute it? what forces the user to follow this route?

Comment (manjunathb4461) [reply venomoid.go]: The assert statements in `TestConfigBuilder_WithAutomaticEnv` show both the way to access the key after reading the env var using the package
`viper.Get` works by default but users need to bind it with `BindEnv`(just like in viper, with env key being in capitals) to be able to read the env var using `config` object as well

Users can directly use `viper.BindEnv` instead but it wouldn't be that nice to import viper just to bind the vars, I can copy over the docs from viper and add them here as it provides the same functionality

**Review by ranglust — CHANGES_REQUESTED** [review]
either bind the BindEnv function into the builder pattern or get rid of it.
also, please update README.md with an example on how this should work

File: venomoid.go (hunk 1 above)
Comment (manjunathb4461) [reply venomoid.go]: Modified it accordingly and also added an example in README

**Pull request merged**